dev/io.openliberty.microprofile.metrics.internal.5.1_fat/publish/servers/NonExistentLibrary/jvm.options

```diff
@@ -0,0 +1 @@
+-Dcom.ibm.tools.attach.enable=yes
```

Comment:
Since, mpMetrics-5.1 is still in beta, shouldn't this file set the BETA JVM prop to be true, for the FATs to run properly?